setup.cfg

```diff
@@ -33,6 +31,5 @@ not_skip = __init__.py
 force_sort_within_sections = true
 sections = FUTURE,STDLIB,INBETWEENS,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
 default_section = THIRDPARTY
-known_first_party = homeassistant,tests
-forced_separate = tests
+known_first_party = custom_components.blueprint 
```

Comment:
Change to tahoma? (and should we keep homeassistant)

Reply:
I’ve removed this. It’s done automatically by isort now.